tests/testthat/testthat.R

```diff
@@ -1,12 +1,15 @@
 library(testthat)
-library(romigami)
+source("/Users/pierre/PycharmProjects/romigami/R/romigami.R")
```

Comment:
do you want to leave it like this? 
this is so you won't have to push to master and install again right?

Reply:
Yes that's exactly why!
And yes I do think it makes more sense for tests to import the romigami functions locally (with `source`) rather than install the package from github master and then import (with `library`).